presto-main/src/main/java/com/facebook/presto/cost/SymbolStatsEstimate.java

```diff
@@ -100,6 +101,26 @@ public double getDistinctValuesCount()
         return distinctValuesCount;
     }
 
+    public SymbolStatsEstimate mapLowValue(Function<Double, Double> mappingFunction)
```

Comment:
maybe we could have our own `double Function#apply(double, double)`?

Reply:
Please explain?